pkgs/applications/version-management/gitlab/data.json

```diff
@@ -1,32 +1,32 @@
 {
   "ce": {
-    "version": "12.1.6",
-    "repo_hash": "0zyqxzyb1m8qa94iiyqf5ivqxrg60y378r94fsixy17fbh4y7sa6",
-    "deb_hash": "14bwcmwar44pjnjr6fdn2h9y92vyfva0kb76d4pz8pqc9pfm2hmg",
-    "deb_url": "https://packages.gitlab.com/gitlab/gitlab-ce/packages/debian/stretch/gitlab-ce_12.1.6-ce.0_amd64.deb/download.deb",
+    "version": "12.3.2",
+    "repo_hash": "1dbxdn2j2y5kzv5qbvpfc0xcpqhgv0ah0wikq33ks0mwavxxzp99",
+    "deb_hash": "1qzh6kn84aiw8dhk8gh0kshv0hq0kvvplfy56d2p6iib51m315s7",
+    "deb_url": "https://packages.gitlab.com/gitlab/gitlab-ce/packages/debian/stretch/gitlab-ce_12.3.2-ce.0_amd64.deb/download.deb",
     "owner": "gitlab-org",
-    "repo": "gitlab-ce",
-    "rev": "v12.1.6",
+    "repo": "gitlab-foss",
```

Comment:
Do we really want to keep separate attrsets? It seems like `gitlab-foss` is just a poorly-synced version of the `gitlab` repo these days...

Reply:
Maybe not? It doesn't seem to be too poorly synced to me, but from what I've gathered, the only difference between them is that the `gitlab` repo has an `ee` directory which is removed from the tree when it's synced to `gitlab-foss`...

Reply:
Well, at least the git repo history is a bit less informative.

Obviously, Nix doesn't care much. However, after reading up on https://about.gitlab.com/2019/02/21/merging-ce-and-ee-codebases/, it should be safe to just remove the `/ee` folder if we're not building the enterprise edition, so I'd advocate for having a conditional step in `postUnpack` depending on the `gitlabEnterprise` flag, and hardcode everything else to https://gitlab.com/gitlab-org/gitlab.

This would greatly simplify data.nix and the updater.

Reply:
Yeah, that states pretty clearly that all we have to do is get rid of the `ee` folder, so I'll look into it. :)